lib/streamlit/__init__.py

```diff
@@ -232,14 +232,13 @@ def set_option(key, value):
 def write(*args, **kwargs):
     """Write arguments to the app.
 
-    This is the swiss-army knife of Streamlit commands. It does different things
-    depending on what you throw at it.
+    This is the Swiss Army knife of Streamlit commands: it does different
+    things depending on what you throw at it. Unlike other Streamlit commands,
+    write() has some unique properties:
 
-    Unlike other Streamlit commands, write() has some unique properties:
-
-    1. You can pass in multiple arguments, all of which will be written.
-    2. Its behavior depends on the input types as follows.
-    3. It returns None, so it's "slot" in the App cannot be reused.
+        1. You can pass in multiple arguments, all of which will be written.
+        2. Its behavior depends on the input types as follows.
+        3. It returns None, so it's "slot" in the App cannot be reused.
```

Comment:
Nit: Are you sure this indentation is right? AFAIK, Markdown bullet lists should align with the parent (which means that in this case the indent should match the left of the paragraph above it).

Both styles seem to compile fine, but the rendered result is slightly different.  Not sure if this was on purpose...

Reply:
For me it was a question of aesthetics.

Without tab:
![without_tab](https://user-images.githubusercontent.com/2762787/81298791-4a2ea180-9043-11ea-9e55-4f8d897c048b.png)

With tab:
![with_tab](https://user-images.githubusercontent.com/2762787/81298801-4e5abf00-9043-11ea-8518-920bb97a2240.png)

Happy to change it back if you like the first version.

Reply:
Reverted it back to less indentation